confgenerator/testdata/valid/linux/all-backward_compatible_with_explicit_exporters/golden_otel.conf

```diff
@@ -344,7 +344,7 @@ processors:
     - gce
 receivers:
   hostmetrics/default__pipeline_hostmetrics:
-    collection_interval: 60s
+    collection_interval: 1m0s
```

Comment:
[Optional] This one is not ideal either, but less annoying than the user-visible sample config change.